scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/scripts/clone-kibana.sh

```diff
@@ -1,9 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
 OWNER=${1:-elastic}
 BRANCH=${2:-master}
 
-echo "Cloning Kibana: $OWNER:$BRANCH"
+echo "Cloning Kibana: ${OWNER}:${BRANCH}"
 
-cd ./tmp
-git clone --depth 1 -b $BRANCH https://github.com/$OWNER/kibana.git
-mv ./kibana/x-pack/legacy/plugins/apm/typings/es_schemas ./apm-ui-interfaces
+cd ./tmp || true
+git clone --quiet --depth 1 -b "${BRANCH}" "https://github.com/${OWNER}/kibana.git"
```

Comment:
- reduce verbose output in the CI.
- fix a lint defect

Reply:
I originally made this git clone a little more complicated than it has to be. What I wanted was a partial checkout of just a specific path (to make it faster). I didn't find any simple ways to do that. 
I don't think I saw any perf improvements by using the `--depth` flag (but not 100%)

Reply:
But you don't have to change this. Just thought I'd mention it.